ceno_emul/Cargo.toml

```diff
@@ -15,6 +15,7 @@ elf = "0.7"
 itertools.workspace = true
 num-derive.workspace = true
 num-traits.workspace = true
+rrs_lib = { package = "rrs-succinct", version = "0.1.0" }
```

Comment:
why not use the standard `rrs_lib`?

Reply:
That's what I tried at first, but then I noticed that Succinct had added support for `ecall` and a few other niceties.